numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1047,8 +1047,8 @@ def add_newdoc(place, name, doc):
 
     References
     ----------
-    .. Wikipedia, "Heaviside step function",
-       https://en.wikipedia.org/wiki/Heaviside_step_function
+    Wikipedia, "Heaviside step function",
+    https://en.wikipedia.org/wiki/Heaviside_step_function
```

Comment:
Just a nit, but according to the [numpydoc docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#references), the references are supposed to be enumerated *and* refer back to the main text. Therefore I vote for one of two courses of action: 1) Remove the references section here entirely or 2) change this to

```suggestion
    .. [1] Wikipedia, "Heaviside step function",
       https://en.wikipedia.org/wiki/Heaviside_step_function
```

And add `[1]_` to L1021.